tests/snapshot.test.ts

```diff
@@ -44,6 +45,136 @@ it('should not change snapshot with assigning same object', async () => {
   expect(snap1).toBe(snap2)
 })
 
+it('converts object properties to readonly', () => {
```

Comment:
Cool. Let's add `describe` something like:
```suggestion
describe('snapshot typing', () => {
  it('converts object properties to readonly', () => {
```

Reply:
Dome.